modules/consensus/synchronize.go

```diff
@@ -10,22 +14,43 @@ import (
 	"github.com/NebulousLabs/bolt"
 )
 
+const (
+	// minNumOutbound is the minimum number of outbound peers required before ibd
+	// is confident we are synced.
+	minNumOutbound = 4
+)
+
 var (
 	// MaxCatchUpBlocks is the maxiumum number of blocks that can be given to
 	// the consensus set in a single iteration during the initial blockchain
 	// download.
 	MaxCatchUpBlocks = func() types.BlockHeight {
 		switch build.Release {
-		case "testing":
-			return 3
+		case "dev":
+			return 10
 		case "standard":
 			return 10
+		case "testing":
+			return 3
+		default:
+			panic("unrecognized build.Release")
+		}
+	}()
+	// sendBlocksTimeout is the timeout for the SendBlocks RPC.
+	sendBlocksTimeout = func() time.Duration {
+		switch build.Release {
 		case "dev":
-			return 10
+			return 5 * time.Minute
```

Comment:
dev machines should assume local networks, no reason to make the timeout a full 5 minutes. 40 seconds should be good.